src/ceph-disk/ceph_disk/main.py

```diff
@@ -323,21 +325,16 @@ def is_systemd():
     """
     Detect whether systemd is running
     """
-    with file('/proc/1/comm', 'rb') as i:
-        for line in i:
-            if 'systemd' in line:
-                return True
-    return False
+    with open('/proc/1/comm', 'rb') as f:
+        return 'systemd' in f.read()
```

Comment:
<code>'systemd' in <b>&lt;bytestring&gt;</b></code> &ndash; please check if this is correct